templates/settings/address-form.php

```diff
@@ -141,6 +141,19 @@
                 </select>
             <?php } ?>
             </div>
-        <?php } ?>
+        <?php
+        }
+        
+        /**
+         * Add vendor address verification templates.
+         * 
+         * @since DOKAN_PRO_SINCE
```

Comment:
As this hook is defined on Dokan lite the since version should be the Dokan lite since. Kindly fix this `@since DOKAN_PRO_SINCE`